### DISCUSSION TITLE
Add animated svg creation

### DISCUSSION THREAD

**Discussion by bandarrex** [photopea/photopea#5513] [Ideas]
Hi  🖐🏼
i am asking the developers to add animated svg creation becausi the web is lacking for good free editors. the web is so live right now and responsive material is better for performance and quality for the content.
i hope to consider this in the near future.

thank you 😎

**Comment by photopea** [comment]
Hi, how should it look like? Do you know any animation editor which you like? Are you sure there do not exist any editors?

**Comment by LazaroOnline** [comment]
This idea seems like a can of worms that could endup becoming a completely new application bigger than Photopea itself.

I would take Adobe AfterEffects as a reference if I am allowed to dream.
Ideally the animations could be exported to the [lottie format](https://lottie.github.io/) to be used in webs and apps.

If it has to be integrated into photopea (and not becoming another web-app) 
then it would make more sense something like the [Photoshop "Timeline" window](https://www.youtube.com/shorts/Z475jHqHNTo)
